lib/lotus/generators/action.rb

```diff
@@ -30,6 +30,16 @@ class Action < Abstract
       # @api private
       DEFAULT_TEMPLATE = 'erb'.freeze
 
+      # Default HTTP method used when generating an action.
+      # @since x.x.x
+      # @api private
+      DEFAULT_HTTP_METHOD = 'get'.freeze
+
+      # Supported HTTP methods when generating an action.
+      # @since x.x.x
+      # @api private
+      AVAILABLE_HTTP_METHODS = %w{get post put patch delete trace}.freeze
```

Comment:
can we use the verbs supported by our router https://github.com/lotus/router/blob/master/lib/lotus/routing/http_router.rb#L10 ?